src/main/java/com/github/lmiljak/dowertefence/OnCommandGameCreator.java

```diff
@@ -0,0 +1,276 @@
+	}
+
+	@Override
+	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+		// Processes the start and accept sub commands
+
+		if (!(sender instanceof Player)) {
+			sender.sendMessage("You must be a Player to send this message");
+			return false;
+		}
+		if (args.length == 0) {
+			// There should always be at least one argument such as "start" or
+			// "accept".
+			return false;
+		}
+
+		if (label.equals(DowerTefence.PARENT_COMMAND)) {
+			switch (args[0]) {
+			case "start":
+				return processStartCommand((Player) sender, removeFirst(args));
+			case "accept":
+				return processAcceptCommand((Player) sender, removeFirst(args));
+			default:
+				return false;
+			}
+		}
+
+		return false;
+	}
+
+	/**
+	 * Takes a String array and removes the first String from it.
+	 * 
+	 * @param stringArray
+	 *            The String array to remove the first element of.
+	 * @return The new String array.
+	 */
+	private String[] removeFirst(String[] stringArray) {
+		String[] result = new String[stringArray.length - 1];
+
+		for (int i = 1; i < stringArray.length; i++) {
+			result[i - 1] = stringArray[i];
+		}
+
+		return result;
+	}
+
+	/**
+	 * Used to process the start command. If the sender of the command invited
+	 * no people, the game is immediately created. In the case that the sender
+	 * did invite people, a PotentialGame will be created until all players have
+	 * accepted.
+	 * 
+	 * @param sender
+	 *            The player that sent the command.
+	 * @param invitedPlayers
+	 *            The players that got invited by sender (the arguments of the
+	 *            command).
+	 * @return True if and only if the command was executed successfully.
+	 */
+	private boolean processStartCommand(Player sender, String[] invitedPlayers) {
+		HashSet<Player> participants = new HashSet<>();
+
+		// Check if the sender wants to play single player, if so then just
+		// create the game with him in it
+		if (invitedPlayers.length == 0) {
+			participants.add(sender);
+			notifyListeners(new StandardGame(participants));
+			return true;
+		}
+
+		participants = invitePlayers(sender, invitedPlayers);
+		if (participants == null) {
+			// The invites have been cancelled and the sender got notified.
+			return true;
+		}
+
+		for (String arg : invitedPlayers) {
+			Player invitedPlayer = Bukkit.getPlayer(arg);
+
+			if (invitedPlayer == null) {
+				sender.sendMessage("One or more of the specified players do not exist or are not online.");
+				return true;
+			}
+
+			participants.add(invitedPlayer);
+		}
+
+		// Creating a new potential game.
+		potentialGames.put(sender, new PotentialGame(participants));
+		sender.sendMessage("Waiting for all invited players to accept.");
+		return true;
+	}
+
+	/**
+	 * Invites an array of players by sending them a message. If one or more of
+	 * the specified players do not exist or are not online, the invites will be
+	 * cancelled and the inviter will be notified.
```

Comment:
Okay let me sketch a situation:
Alice, Bob, Chris and Dick are online.

Alice wants to invite all of them for a game so she executes the following command: "/dt start Bob Chtis Dick". Oh no! She misspelled Chris. In your case, Bob and Dick will still be invited and Alice will get the message "Not all of the specified players have been invited, as some do not exist or are not online.". Before Alice realises her mistake though, Bob and Dick could already have accepted the invite and the game has started without Chris.
Now the other three have to go through all the trouble of cancelling the game so that they can start it again with Chris.